### PR TITLE
feat: Add a 'new' badge to newly published courses

### PR DESCRIPTION
**Pull request by josefacastillo**
info: I still need to get publichedAt into SubcourseData, didnt have more time

Update: done :P

### CODE REVIEW

**Review by Jonasdoubleyou — APPROVED** [review]
nice! :)

File: src/widgets/AppointmentCard.tsx [hunk 1]
```diff
@@ -50,6 +51,7 @@ type Props = {
     description: string;
     maxParticipants?: number;
     participantsCount?: number;
+    publishedAt?: Date;
```
Comment (Jonasdoubleyou) [inline src/widgets/AppointmentCard.tsx:54]: As it is an optional property, do we fetch it in all relevant GraphQL queries?

Comment (josefacastillo) [reply src/widgets/AppointmentCard.tsx:54]: Good question. I wasnt sure about that actually, how can I find out if is always there?

Comment (Jonasdoubleyou) [reply src/widgets/AppointmentCard.tsx:54]: You could make it mandatory and see if it still compiles :)

Comment (Jonasdoubleyou) [reply src/widgets/AppointmentCard.tsx:54]: Ah, nevermind - it's also nullable in the model. Not sure but : Date | null might work (excluding undefined)

Comment (Jonasdoubleyou) [reply src/widgets/AppointmentCard.tsx:54]: Hm, for nullable properties that is tricky as we mix null and undefined ... I think it is fine if you check (Dashboard + Group Page) x (Pupil + Student) :)

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**